src/components/common/Tile/PersonTile/styled.jsx

```diff
@@ -0,0 +1,72 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+
+  width: 208px;
+  height: auto;
+  padding: 16px;
+  box-shadow: ${({ theme }) => theme.properties.tile.boxShadow};
+  gap: 12px;
+
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    justify-self: center;
+    width: 136px;
+    height: 245px;
+    gap: 8px;
+  }
+`;
+
+export const ImageWrapper = styled.div`
+  display: flex;
+  border-radius: 5px;
+  width: 100%;
+  height: 75%;
+  background-color: ${({ theme }) => theme.colors.tile.imageWrapper};
+  align-self: center;
+  justify-content: center;
+
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    width: 114px;
+    min-width: 114px;
+    height: 169px;
+    min-height: 169px;
+  }
+`;
+
+export const Image = styled.img`
+  object-fit: cover;
+  object-position: center;
+  border-radius: 5px;
+  height: 100%;
+  width: 100%;
+`;
+
+export const Character = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+  margin: 10px 0;
+  height: 25%;
+`;
+
+export const Name = styled.p`
```

Comment:
I observerd weird think about displaying name.
![image](https://github.com/jakub-bartoszek-developer/Movie-browser/assets/113419864/b0f347b0-e9d1-4ba6-bcb5-3331cd6ec402)
I think its because of the percentage height. I mean 75% for image and 25% for character. Try to get rid of that.

Reply:
that was Simon's solution. I removed it and set justify-content in the centre. looks ok now